### PR TITLE
Xcode 13/14 binary compatibility fix

### PR DESCRIPTION
**Pull request by CedricGuillemet**

### CODE REVIEW

**Comment by ryantrem** [comment]
Did this actually fix BRN built with xcode 14 and used in xcode 13, or yet to be tested?

**Comment by CedricGuillemet** [comment]
> Did this actually fix BRN built with xcode 14 and used in xcode 13, or yet to be tested?

this is yet to be tested.

**Comment by CedricGuillemet** [comment]
Tested OK on iOS with xCode14!

**Comment by CedricGuillemet** [comment]
Tested OK with xCode13

File: Package/iOS/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.13.2)
 project(ReactNativeBabylon)
+
+# Clang will by default emit objc_msgSend stubs in Xcode 14, which ld from earlier Xcodes doesn't understand.
+# We disable this by passing -fno-objc-msgsend-selector-stubs to clang.
```
Comment (bghgary) [inline Package/iOS/CMakeLists.txt:5]: Maybe a pointer to this link for additional info: https://stackoverflow.com/questions/73983452/xcode-14-xcode-13-incompatibility

**Review by bghgary — APPROVED** [review]

**Review by ryantrem — APPROVED** [review]

**Pull request merged**